src/pages/user/authority/AuthorityModal.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import { Checkbox, FormControlLabel } from '@material-ui/core';
+
+import ButtonModal from '../../../components/ButtonModal';
+import useLocalStorage from '../../../utils/hooks/useLocalStorage';
+import { ConnectAuthority } from './ConnectAuthority';
+
+const StyledCheckbox = styled(FormControlLabel)`
+  margin-top: 2rem;
+  align-self: center;
+`;
+
+interface AuthorityModalProps {
+  showModal: boolean;
+}
+
+const AuthorityModal: React.FC<AuthorityModalProps> = ({ showModal }) => {
+  const { t } = useTranslation();
+  const [checked, setChecked] = useState(false);
+  const [doNotShowAgain, setDoNotShowAgain] = useLocalStorage('doNotShowAgain', { connectAuthorityModal: false });
+  const localStorageRef = useRef(!doNotShowAgain.connectAuthorityModal);
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
mer beskrivende navn på denne og `checked`? Jeg tenkte første at dette var handleChange for når man velger en autoritet, og at `checked` var om en autoritet var valgt